openmp/libomptarget/test/offloading/fortran/target_map_common_block.f90

```diff
@@ -26,11 +30,14 @@ subroutine check_device
   !$omp target map(tofrom:devices)
     devices(2) = omp_get_device_num()
   !$omp end target
+  print *, omp_get_num_devices()
+  !CHECK: [[ND:[0-9]+]]
+  print *, omp_get_default_device()
+  !CHECK: [[DD:[0-9]+]]
+  !CHECK: devices: [[ND]] [[DD]]
   print *, "devices: ", devices
 end subroutine check_device
 
-!CHECK: devices: 1 0
-
 !-----
 
 subroutine commonblock_simple_with_implicit_type
```

Comment:
Can you remove the explicit mapping from this test? I guess we want to check implicit mapping here.